Features/CopyMoveDelete/CopyContent.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Copy content
+    In order to copy objects
+    As an Editor user
+    I need to be able to copy an object that I am viewing
+
+    Background:
+        Given I am logged in as an Editor in PlatformUI
+
+    ##
+    #Copy objects to valid locations
+    ##
+    @javascript
+    Scenario: Copy one object without children objects
```

Comment:
children aren't supposed to matter in copy, are they ? Children aren't copied by this operation, that would be subtree copy (unless of course the feature has evolved, in which case ignore).